refinery/core/views.py

```diff
@@ -69,6 +69,14 @@ def home(request):
     )
 
 
+def home_v2(request):
+    return render_to_response(
```

Comment:
[The `context_instance` kwarg has been deprecated in Django 1.8](https://docs.djangoproject.com/en/1.10/releases/1.8/#dictionary-and-context-instance-arguments-of-rendering-functions)

I don't think you need the context of the request passed along to the template either. The following should be fine:

```suggestion
    return render('core/home_v2.html')
```